#!/usr/bin/env python
PACKAGE = "tuw_gazebo_plugins"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("fps", int_t, 0, "update rate of the detection", 10, 0, 50)

gen.add("p_detection", double_t, 0, "probability that a cone will be detected", 0.7, 0, 1)

gen.add("sig_range", double_t, 0, "Scale of measurement range noise", 1, 0, 10)
gen.add("sig_bearing", double_t, 0, "Scale of measurement bearing noise", 1, 0, 10)
gen.add("sig_correlation", double_t, 0, "Scale of the correlation between range and measurement noise", 1, 0, 10)

exit(gen.generate(PACKAGE, "tuw_gazebo_plugins", "ConeDetectionSim"))
